config/locales/es/es.yml

```diff
@@ -116,6 +116,7 @@ es:
         secondary_agent: "Agente Secundario"
         signing_date: "Fecha de Inicio"
         status: "Estado"
+        health_check: "Chequeo de Salud"
```

Comment:
You don't need to add Spanish translations. Translations are managed in Transifex. Only adjust English translations, otherwise your changes in the Spanish files may be overridden.